public/js/main.js

```diff
@@ -160,26 +167,43 @@ $(document).on('ready', function() {
     });
   }
 
+  var userBoxIndex = 0;
+  function handleUserEvent(data) {
+    var container = $('.slide-feed');
+    var children = container.children();
+    if(userBoxIndex >= children.length){
+      userBoxIndex = 0;
+    }
+    var name = data.name || "Another Do-er";
+    switch(data.activity) {
+      case "user_register":
+        updateBox('box--register', '<p><strong>' + name + "</strong> created an account!</p>", container, userBoxIndex);
```

Comment:
Consider adding the markup to some kind of template file? It feels odd to see HTML markup in code.